Monika After Story/game/0utils.rpy

```diff
@@ -647,6 +703,56 @@ python early in mas_utils:
         return False
 
 
+    def _get_version(ver_str, nums_only, split_nums=False):
+        """
+        Gets the version from a ver str
+
+        IN:
+            ver_str - version string to get version from
+            nums_only - True to only get ver numbers, False to get full ver
+
+        RETURNS: version str
+        """
+        if nums_only:
+            ver_nums = ver_str.partition("-")[0]
+        
+            if split_nums:
+                return ver_nums.split(".")
+
+            return ver_nums
+
+        return ver_str
```

Comment:
I'm being picky, but this is kind of odd function.

You don't seem to be using `nums_only` as `False` at all. But I also don't think it'd ever make sense to as it'll just return the string you passed in. So I think it'd be cleaner if we just removed it.

I think we should also document the `split_nums` param because it changes the output dramatically (you can get either a `str` or a `list` of `str`s.